modules/rostests/winetests/kernel32/process.c

```diff
@@ -295,6 +295,21 @@ static void WINETEST_PRINTF_ATTR(2,3) childPrintf(HANDLE h, const char* fmt, ...
     WriteFile(h, buffer, strlen(buffer), &w, NULL);
 }
 
+static UINT
```

Comment:
Will this changes make sense for Wine at some point (i.e. succeed on all Windows versions), or should this eventually become an apitest?

Reply:
As of now, this can only run in interactive mode since it spawns dialogs from csrss